### PR TITLE
reworked panels and promos to cater for nhsuk v4

### LINKED ISSUE #37: Hero block on non NHS themes
The jquery that moves the hero block to the top of the page and suppresses page title doesnt work on non Nightingale themes. This is because the selectors used are theme specific. This will not be an easy fix as will need to identify selectors that are global across all themes...not even sure if that is possible.
nhsblocks.php lines 286 onwards:
```
function nhsblocks_hero_footer() {
	echo "<script>
	    const heroBlock = document.querySelector('.wp-block-nhsblocks-heroblock');
	    if ( ( heroBlock ) ) { 
	        matches = heroBlock.matches ? heroBlock.matches('.wp-block-nhsblocks-heroblock') : heroBlock.msMatchesSelector('.wp-block-nhsblocks-heroblock');
		    if ( matches === true ) {
			    const mainContent = document.querySelector( '#maincontent' );
			    const contentInner = document.querySelector( '#contentinner' );
			    const wholeDoc = document.querySelector( 'body' );
			    const breadCrumb = document.querySelector( '.nhsuk-breadcrumb' );
			    const articleTitle = document.querySelector( '.entry-header' );
			    mainContent.insertBefore( heroBlock, contentInner );
			    articleTitle.style.display = 'none';
			    mainContent.style.paddingTop = '0';
			    heroBlock.style.marginBottom = '70px';
			    if ( breadCrumb ) { 
			    	wholeDoc.removeChild( breadCrumb );
			    }
		    }
	    }
```

### PR DESCRIPTION
**Pull request by tblacker7**
NHSUK has deprecated panels and promos, and made both the same new element - card. Promo cards are clickable, otherwise identical. However, we have historic blocks on sites and through the WordPress plugin installations which use the old markup.

This fix adds the old styles as legacy style, but also modifies the markup on any panel and promo on edit of page/post (this fires automagically when a page is edited, regardless of whether the component is modified) by changing the edit and save output and including the original markup in a deprecated statement.

I can see no way of combining the two elements into a single new one without breaking the content that was created before this change. Therefore, I have intentionally retained the naming of panel and promo. Fixes #37

### CODE REVIEW

File: build/index.asset.php [hunk 1]
```diff
@@ -1 +1 @@
-<?php return array('dependencies' => array('lodash', 'wp-element', 'wp-polyfill'), 'version' => 'b1ed361d4d0e7f58a477b80c455cde99');
\ No newline at end of file
+<?php return array('dependencies' => array('lodash', 'wp-element', 'wp-polyfill'), 'version' => 'a5a3a61e925acbbba98344b12938e3d4');
```
Comment (Nick-Summerfield) [inline build/index.asset.php]: No new line here.

Comment (tblacker7) [reply build/index.asset.php]: This is a system generated file. No need for newlines - its best practice but not essential. And in this instance, not possible to force.

**Comment by marcinca** [comment]
Not approving it yet as I found some problem with Heading Block - @tblacker7 investigating
image

image

**Comment by tblacker7** [comment]
> Not approving it yet as I found some problem with Heading Block - @tblacker7 investigating
> image
> 
> image

unable to reproduce this 🤔  Both Nick and I tested with quote and heading blocks, and neither had this show up...

**Review by marcinca — APPROVED** [review]

**Pull request merged**